### ISSUE TITLE
933150: New tests failing

### ISSUE THREAD

**Issue by lifeforms** [SpiderLabs/owasp-modsecurity-crs#1271]
I've added two tests for rule 933150 in #1268. The tests succeeded then, but they seem to be failing now for an unknown reason:

```
util/regression-tests/CRS_Tests.py::test_crs[ruleset240-933150.yaml -- 933150-15] FAILED
util/regression-tests/CRS_Tests.py::test_crs[ruleset241-933150.yaml -- 933150-16] FAILED
```

https://travis-ci.org/SpiderLabs/owasp-modsecurity-crs/builds/472175453

Tests are also succeeding when I run them locally in my owasp/modsecurity:v2-ubuntu-apache docker...

Link to the two changed tests: https://github.com/SpiderLabs/owasp-modsecurity-crs/blob/ce524ee1342d0a114d9172a4f66a7fca1e866078/util/regression-tests/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933150.yaml#L208

**Comment by dune73** [comment]
Confirm. Tests run fine locally here too.

**Comment by dune73** [comment]
@csanders-git : could you take a look at travis? What other options do we have? We need to fix this quickly. Development is has picked up some steam and not being able to do CI would kill our enthusiasm.

**Comment by lifeforms** [comment]
The problem was found and fixed by @fgsch in #1273.
